messages/messages.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2016 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package messages provides functions for validating payloads from GitHub Webhooks.
+// GitHub docs: https://developer.github.com/webhooks/securing/#validating-payloads-from-github
+package messages
```

Comment:
depending on how much we can simplify the number of exported methods, we might be able to just move this into the main github package.